src/style.css

```diff
@@ -1,3 +1,69 @@
-body {
+body,
+html {
+  padding: 0;
   margin: 0;
+  box-sizing: border-box;
+  font-family: Roboto, sans-serif;
```

Comment:
Remember to use fallback fonts - alternative font-family in case the main one doesn't work.